controller_interface/src/controller_interface_base.cpp

```diff
@@ -233,4 +242,19 @@ void ControllerInterfaceBase::wait_for_trigger_update_to_finish()
     async_handler_->wait_for_trigger_cycle_to_finish();
   }
 }
+
+std::string ControllerInterfaceBase::get_name() const { return get_node()->get_name(); }
```

Comment:
Should we just move this one liner to the header?

Reply:
Sure. I can do it.